SparkPost/2web2ui
# FE-1167: Secure webui cookies

[FE-1167](https://sparkpost.atlassian.net/browse/FE-1167)

### What Changed
 - Set [secure](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#SameSite_attribute#Restrict_access_to_cookies) to "true" and [sameSite](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#SameSite_attribute) to "strict" attributes for `auth` and `website_auth` cookies
 - Had to modify the test configuration to not require secure cookies and fix blocklist spec

### How To Test

_**You will need to setup [charger](https://github.com/SparkPost/charger) to verify this locally.**_

 1. Read https://github.com/js-cookie/js-cookie and https://web.dev/samesite-cookies-explained
 1. Visit http://app.sparkpost.test/auth
 1. Open "Application" tab in devtools 
 1. Clear cookies
 1. Login
 1. Confirm both cookies are created with secure and sameSite attributes are set
 1. Open "http://www.sparkpost.test"
 1. Open "Application" tab in devtools 
 1. Confirm `website_auth` cookie is set
